demo/components/selection-list-demo.tsx

```diff
@@ -0,0 +1,101 @@
+import React = require('react');
+import {observable} from 'mobx';
+import {observer} from 'mobx-react';
+import {SelectionList, ItemRendererProps, divider} from "../../src/components/selection-list";
```

Comment:
I decided not to re-export SelectionList from `src/components/index`, because then I'd have to prefix everything with `SelectionList-`, and the names would be very verbose, e.g. `SelectionListItemRendererProps`.